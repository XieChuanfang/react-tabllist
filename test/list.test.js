import React from 'react'
import ReactTabllist from '../src/list'
import Enzyme from './enzyme.config'

it('exports modules correctly', () => {
  expect(ReactTabllist).toMatchSnapshot()
})
