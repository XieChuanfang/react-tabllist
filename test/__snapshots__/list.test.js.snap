// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports modules correctly 1`] = `[Function]`;
